rebar.config

```diff
@@ -1,3 +1,3 @@
 {deps, [
-    {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "v2.8.0"}}}
+    {jsx, ".*", {git, "https://github.com/talentdeficit/jsx.git", {tag, "v2.8.0"}}}
```

Comment:
You're right. I didn't thought about it. Maybe a note on the README about the requirement to use `rebar3` would be useful. Or maybe, if you accept the patch you can make your `rebar.config` compatible with both `rebar2` and `rebar3` versions.